### ISSUE TITLE
simple net usage has unexpected results

### ISSUE THREAD

**Issue by t4k** [caltechlibrary/commonpy#2]
## Describe the bug

With just a method and a url for net(), there is no result and a ValueError. I am able to get a result with cURL.

## To Reproduce

```
from commonpy.network_utils import net
print(net('get', 'https://caltech.tind.io/search?ln=en&p=barcode%3A+35047019492099&of=xm'))
```
result:
`(None, ValueError('httpx.Timeout must either include a default, or set all four parameters explicitly.'))`

## Environment (please complete the following information)

 - Operating system: macOS 10.15.7
 - Version of Python: 3.9.1

## Additional context

cURL can return the expected XML:
`curl -H "Accept: application/xml" -H "Content-Type: application/xml" -X GET "https://caltech.tind.io/search?ln=en&p=barcode%3A+35047019492099&of=xm"`

**Comment by mhucka** [comment]
I can reproduce this. Looks like a bug. Will investigate.

**Comment by mhucka** [comment]
The interface for httpx Timeout changed last August (https://github.com/encode/httpx/releases/tag/0.14.0) and I guess I must have been testing against an older version of the library. I'll fix it and make a new release of CommonPy today.

**Comment by mhucka** [comment]
This should be fixed in version 1.3.0, just released.
